Makefile.am

```diff
@@ -240,13 +240,12 @@ lint: flake8 cpplint
 .PHONY : flake8
 flake8: Makefile.am
 if FOUND_FLAKE8
-	flake8
+	flake8 --statistics --count
```

Comment:
I assume this doesn't hide the individual faults?

Reply:
No, does not hide any information.